src/Components/Search/NewSearch/SearchInputContainer.tsx

```diff
@@ -0,0 +1,52 @@
+import { LabeledInput, Clickable, LabeledInputProps } from "@artsy/palette"
+import { themeGet } from "@styled-system/theme-get"
+import { isEmpty } from "lodash"
+import * as React from "react"
+import styled from "styled-components"
+import SearchIcon from "@artsy/icons/SearchIcon"
+
+const SearchButton = styled(Clickable)`
+  width: 18px;
+  height: 18px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 50%;
+
+  &:focus {
+    background: ${themeGet("colors.purple100")};
```

Comment:
Purple!?

Reply:
Just copied it over from [here](https://github.com/artsy/force/blob/e1566ca9ff8e117d7c350ecb5900e8ef0bcbdd73/src/Components/Search/SearchInputContainer.tsx#L17)

Reply:
removed